### DISCUSSION TITLE
Deployment from docker compose to k8's

### DISCUSSION THREAD

**Discussion by mdominguezd** [developmentseed/eoAPI#195] [Q&A]
Hey hey,

I am quite new using Docker and Kubernetes. I tried deploying the eoApi in a Kubernetes cluster with the helm chart provided, but already had some issues. Firstly, the raster service deployed constantly showed error 503: service unavailable. Moreover, I was not able to ingest data to the stac api created. I was trying to do it from within the stac pod, but the version of pypgstac did not allowed it. Tried installing psycopg, but then ended up spending an hour trying to solve pip dependency issues.

On the other hand, I have worked with the docker compose and by editing some small things everything works pretty well.

Do you know if there's a way to easily deploy in a Kubernetes cluster directly using the docker compose file (and an override with some edits)? 

Any tips are highly appreciated

**Comment by vincentsarago** [comment]
@mdominguezd can you try opening an issue with a bit more details on https://github.com/developmentseed/eoapi-k8s

> On the other hand, I have worked with the docker compose and by editing some small things everything works pretty well.

is there anything you changed that could be useful to others?
